app/src/main/java/com/example/tour_a_versity_v3/MapsActivity.kt

```diff
@@ -58,14 +60,8 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWi
         )
 
         // marker for CECH
-        val cechLat = 39.130315
-        val cechLng = -84.518680
-        val cechLatLng = LatLng(cechLat, cechLng)
-        cechMarker = map.addMarker(MarkerOptions()
-                .position(cechLatLng)
-                .title("CECH")
-                .snippet("College of Education, Criminal Justice, Human Services, and IT")
-        )
+        createCechMarker()
```

Comment:
The `onMapReady` method is already 100 lines long, and it will only get longer if logic is added to it. I would really recommend creating smaller private methods that do specific jobs, and the `onMapReady` method should simply call these methods. Otherwise it would be very difficult to locate code and debug in the future.

Reply:
Making private methods is a good idea, to reduce the size of this method.  Even better idea: read data as data; don't name variables after data.  So, createCechMarker() should just be createMarker(), and the building/college can come from data.